src/server/pps/pretty/pretty.go

```diff
@@ -286,6 +286,9 @@ ParallelismSpec: {{.Details.ParallelismSpec}}
 {{ if .Details.SidecarResourceLimits }}SidecarResourceLimits:
   CPU: {{ .Details.SidecarResourceLimits.Cpu }}
   Memory: {{ .Details.SidecarResourceLimits.Memory }} {{end}}
+{{ if .Details.SidecarResourceRequests }}SidecarResourceRequests:
```

Comment:
We should print requests before limits, to be consistent with the non-sidecar stuff above.